src/main/webapp/app/shared/monaco-editor/model/actions/adapter/monaco-text-editor.adapter.ts

```diff
@@ -152,8 +152,8 @@ export class MonacoTextEditorAdapter implements TextEditor {
         return this.editor.getDomNode() ?? undefined;
     }
 
-    typeText(text: string) {
-        this.editor.trigger('MonacoTextEditorAdapter::typeText', 'type', { text });
```

Comment:
Typing is no longer sufficient to trigger completions, so I added a method that explicitly brings up the suggest widget